match/lib/match/runner.rb

```diff
@@ -71,6 +71,51 @@ def run(params)
       GitHelper.clear_changes
     end
 
+    def import_certificate(args, params)
+      UI.user_error!("It's not allowed to import your certificate because you enabled `readonly` option.") if params[:readonly]
+      UI.warning("Enterprise profiles are currently not officially supported in _match_, you might run into issues") if params[:type] == "enterprise"
+
+      params[:import_certificate] = args[0] if params[:import_certificate].nil?
+      UI.user_error!("Missing path to the certificate to import") if params[:import_certificate].nil?
+
+      params[:import_certificate_password] = args[1] if params[:import_certificate_password].nil?
```

Comment:
@DanToml not sure what do you mean. Anyway, fixed all other comments, thanks! Let me know if you need something else :)